R/texreg.R

```diff
@@ -1925,6 +1925,467 @@ print.texregTable <- function(x, ...) {
   cat(x, ...)
 }
 
+#' Create coefficient plots from statistical model output using \pkg{ggplot2}.
+#'
+#' Create coefficient plots of \R regression output using \pkg{ggplot2}.
+#'
+#'  The \code{plotreg} function produces coefficient plots (i.e., forest plots 
+#'  applied to point estimates and confidence intervals) and works much like 
+#'  the \code{\link{screenreg}}, \code{\link{texreg}}, \code{\link{htmlreg}}, 
+#'  \code{\link{matrixreg}} and \code{\link{wordreg}} functions. It accepts a 
+#'  single or multiple statistical models as input and internally extracts the
+#'  relevant data from the models. If confidence intervals are not defined 
+#'  in the extract method of a statistical model (see \link{extract} and 
+#'  \link{extract-methods}), the default standard errors are converted to 
+#'  confidence intervals. Most of the arguments work either like in the
+#'  \code{\link{screenreg}}, \code{\link{texreg}}, and \code{\link{htmlreg}} 
+#'  \code{\link{matrixreg}} and \code{\link{wordreg}} functions.It is 
+#'  possible to display the plots in two ways: using the \code{type = "facet"} 
+#'  option, one forest plot applied to point estimates and confidence intervals 
+#'  will be visualised in case there is only one model. If there is more than 
+#'  one model each one will be plotted next to the other; using the 
+#'  \code{type = "forest"} option, coefficients from one or more models will 
+#'  be grouped together and displayed as a forest plot. 
+#'
+#' @param custom.title With this argument, a replacement text for the \code{ggtitle} 
+#'   that provides a title above the diagram can be provided. If an empty character 
+#'   object is provided (\code{custom.note = ""}), the note will be omitted completely.
+#' @param override.pval Replacement list of p-value vectors.
+#' @param ci.level If standard errors are converted to confidence intervals
```

Comment:
Inherit `ci.level` as well?

Reply:
`ci.level` is only in `plotreg `since the old version. In order to inherit it, I should change its name...

Reply:
Ah right, don't change it then.